web/dist/incuna-forms.js

```diff
@@ -11,17 +11,77 @@ var angular = exports.angular = window.angular;
 
 var _libraries = require('./libraries.js');
```

Comment:
Why use `_` at the start of these vars? You don't seem to have any conflicting variable names?